pre-commit

```diff
@@ -0,0 +1,56 @@
+#! /bin/bash
+#
+# This script will pass all our added, copied, modified or renamed files to the
+# some commands, which will check them and report back errors
+#
+# @version	0.0.1
+# @author	Tijs Verkoyen <tijs@sumocoders.be>
+
+# initialize vars
+errors=false
+
+# Check if we're on a semi-secret empty tree
+if git rev-parse --verify HEAD
+then
+    against=HEAD
+else
+    # Initial commit: diff against an empty tree object
+    against=4b825dc642cb6eb9a060e54bf8d69288fbee4904
+fi
+
+# Stash changes, we only want to check what's really in the commit
+git stash -q --keep-index
+
+# fetch all changed files
+files=$(git diff-index --name-only --diff-filter=ACMR $against)
+
+if [ -n "$files" ]; then
+    currentPath=$(pwd)
+    paths=()
+
+    for file in $files
+    do
+        paths+=($currentPath/$file)
+    done
+
+    # run our sumo hook check command
+    output=$(sumo hooks check ${paths[@]})
+    echo -e "$output"
+
+    # check if there are errors
+    if [[ $output =~ .*Error:\ .* ]]
+    then
+        errors=true;
+    fi
+fi
+
+# Let's pop our stash. We don't want to lose anything
+git stash pop -q
+
+# if we have errors, exit with 1
+if [ $errors = true ]
+then
+    echo ""
+    echo "If you are sure you want to commit those files, use --no-verify option"
```

Comment:
I'd also add the -n option. so `If you are sure you want to commit those files, use --no-verify (-n) option"`